src/extensions/extensionsApi.ts

```diff
@@ -7,12 +7,24 @@ import * as api from "../api";
 import { logger } from "../logger";
 import * as operationPoller from "../operation-poller";
 import { FirebaseError } from "../error";
-import { RegistryLaunchStage, Visibility } from "../extensions/extensionsHelper";
 
 const VERSION = "v1beta";
 const PAGE_SIZE_MAX = 100;
 const refRegex = new RegExp(/^([^/@\n]+)\/{1}([^/@\n]+)(@{1}([a-z0-9.-]+)|)$/);
 
+export enum RegistryLaunchStage {
```

Comment:
Can this be just `LaunchStage`?

If we need other LaunchStages we can always do `import as`

Reply:
Named it this way to match the field/enum on the backend, so I'd prefer to keep it.

Reply:
ok didn't know backend is named this way, sg to keep it